src/components/FilterDropdown/LogicSelector.jsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+
+import { LeftArrow } from "neetoicons";
+import { Button, DatePicker, Input, Radio } from "neetoui";
+
+import { ATTRIBUTE_TYPES, LOGIC_LABELS } from "constants";
+
+const LogicSelector = ({ node, setNode }) => {
+  return (
+    <>
```

Comment:
Prefer implicit returns
```suggestion
const LogicSelector = ({ node, setNode }) => (
  <>
```